app/js/components/orderEntry/PastOrders.jsx

```diff
@@ -36,17 +36,15 @@ export class PastOrders extends React.Component {
             <thead>
               <tr>
                 <th className="w-145-px">Date</th>
-                <th className="w-120-px">Status</th>
                 <th>Details</th>
```

Comment:
The past drug orders shown here has two table headers while the active drug orders table has three column headers. Would this implementation not result in an inconsistent UI?

Reply:
While @fuchodeveloper  the last column for past drug orders had no meaning.

Reply:
We agreed on this during the demo